File: node/3465.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");
let [a, b, c] = lines.shift().split(" ").map(Number);
const p = (a + b + c) / 2;
const area = Math.sqrt(p * (p - a) * (p - b) * (p - c));
console.log(area.toFixed(3) + " m2");

File: go/3465.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var a, b, c float64

	fmt.Scanf("%f %f %f", &a, &b, &c)

	p := (a + b + c) / 2

	area := math.Sqrt(p * (p - a) * (p - b) * (p - c))

	fmt.Printf("%.3f m2\n", area)
}
